src/components/Home.js

```diff
@@ -1,31 +1,46 @@
 import { useSelector, useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
 import Book from './Book';
-import { addBook } from '../redux/books/booksSlice';
+import { newBook, fetchBooks } from '../redux/books/booksSlice';
 
 const Home = () => {
-  const books = useSelector((state) => state.book.books);
-
   const dispatch = useDispatch();
-
+  const books = useSelector((state) => state.book.books);
   const handleSubmit = (e) => {
+    dispatch(fetchBooks());
+    const itemId = uuidv4();
+    // const categories = '';
     e.preventDefault();
-    const title = e.target[0].value;
-    const author = e.target[1].value;
-    dispatch(addBook({ title, author }));
+    const title = e.target[0].value.trim();
+    const author = e.target[1].value.trim();
+    const categories = e.target[2].options[e.target[2].selectedIndex].text;
+    dispatch(newBook({
+      item_id: itemId, title, author, category: categories,
+    }));
     e.target[0].value = '';
     e.target[1].value = '';
   };
 
   return (
     <div className="home">
-      <h1>Welcome to Bookstore</h1>
       {books.map((book) => (
-        <Book key={book.item_id} title={book.title} author={book.author} id={book.item_id} />
+        <Book
+          key={book.Id}
+          title={book.title}
+          author={book.author}
+          Id={book.Id}
+          category={book.category}
+        />
       ))}
       <form onSubmit={handleSubmit}>
         <input type="text" placeholder="title" required />
         <input type="text" placeholder="author" required />
-        <button type="submit">Add Book</button>
+        <select name="choice" className="choice">
+          <option value="first">Action</option>
+          <option value="second" selected>Fiction</option>
+          <option value="third">Economy</option>
+        </select>
+        <button type="submit" className="Rectangle-2">Add Book</button>
       </form>
     </div>
   );
```

Comment:
- [ ] It is required to produce a close replica of zeplin design. On the zeplin design, there're only two input fields which are for title and category. Kindly remove the author input and leave only the title and category input fields. You can create an array of names that would be used inside a function that can be called to generate random categories when a book is created.

Sample: 
```
   const randomAuthors = [
      'Author One',
      'Author Two',
      'Author Three',
  ];

  const getRandomAuthor = () => {
    const randomIndex = Math.floor(Math.random() * randomAuthors.length);
    return randomAuthors[randomIndex];
  };
```

### Your design
![your-design-2](https://user-images.githubusercontent.com/26597432/232205281-2d671f3f-5772-45fc-8129-865f78f87421.png)


### Zeplin design
![zeplin-2](https://user-images.githubusercontent.com/26597432/232203201-3e1628eb-514f-4ebc-8740-d3394cadc850.png)